main/abseil-cpp/template.py

```diff
@@ -1,14 +1,16 @@
 pkgname = "abseil-cpp"
 pkgver = "20230802.0"
-pkgrel = 0
+pkgrel = 1
 build_style = "cmake"
 configure_args = [
     "-DCMAKE_CXX_STANDARD=17",
     "-DBUILD_SHARED_LIBS=ON",
     "-DABSL_PROPAGATE_CXX_STD=ON",
+    "-DABSL_USE_EXTERNAL_GOOGLETEST=ON",
+    "-DABSL_BUILD_TESTING=ON",
 ]
 hostmakedepends = ["cmake", "ninja", "pkgconf"]
-makedepends = ["linux-headers"]
+makedepends = ["linux-headers", "gtest-devel"]
```

Comment:
the added absl test libraries from build_testing link to gtest which means gtest gets added as a runtime dependency even though nothing actually uses it after build-time check. would be nice to have a subpackage for those specific extra libs that link to it (they'll still be deps of -devel)